### ISSUE TITLE
Build sfproj solution fails on VSTS.  The OutputPath property is not set for project '.csproj'.  

### ISSUE THREAD

**Issue by denizusky** [Azure/service-fabric-issues#1313]
I've set up a new build pipeline on VSTS using the "Azure Service Fabric application" template however I'm not able to get past the "Build **SfProj** solution step" which is complaining (for all the class libraries added to the project) about "The OutputPath property is not set for project '########.csproj'." .

I'm using VS2017 Hosted Agent Queue, and the solution is building/running fine locally with my VS 2017 15.8.4. 

From build log:
```
2018-10-12T19:12:04.8155621Z **##[section]Starting: Build solution **\*.sfproj**
2018-10-12T19:12:06.4620390Z ##[command]"C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\msbuild.exe" "D:\a\1\s\Server\###.ServiceFabric\###.ServiceFabric.sfproj" /nologo /nr:false /dl:CentralLogger,"D:\a\_tasks\VSBuild_71a9a2d3-a98a-4caa-96ab-affca411ecda\1.126.0\ps_modules\MSBuildHelpers\Microsoft.TeamFoundation.DistributedTask.MSBuild.Logger.dll";"RootDetailId=68f12243-197b-4451-abc5-780d0bda4580|SolutionDir=D:\a\1\s\Server\###.ServiceFabric"*ForwardingLogger,"D:\a\_tasks\VSBuild_71a9a2d3-a98a-4caa-96ab-affca411ecda\1.126.0\ps_modules\MSBuildHelpers\Microsoft.TeamFoundation.DistributedTask.MSBuild.Logger.dll" /t:Package /p:PackageLocation=D:\a\1\a\applicationpackage /p:platform="x64" /p:configuration="Release" /p:VisualStudioVersion="15.0" /p:_MSDeployUserAgent="VSTS_c5933c12-75fe-4fa6-8e2b-34f152b6d5e5_build_53_0"

##[error]C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(780,5): 
Error : The OutputPath property is not set for project '########.csproj'.  
Please check to make sure that you have specified a valid combination of Configuration and Platform for this project.  
Configuration='Release'  Platform='x64'.  
You may be seeing this message because you are trying to build a project without a solution file, 
and have specified a non-default Configuration or Platform that doesn't exist for this project.
```

.sfproj file
```
 <Import Project="..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.7\build\Microsoft.VisualStudio.Azure.Fabric.Application.props" Condition="Exists('..\packages\Microsoft.VisualStudio.Azure.Fabric.MSBuild.1.6.7\build\Microsoft.VisualStudio.Azure.Fabric.Application.props')" />
 <PropertyGroup Label="Globals">
    ..
	<ProjectVersion>2.3</ProjectVersion>
    <MinToolsVersion>1.5</MinToolsVersion>
    <SupportedMSBuildNuGetPackageVersion>1.6.7</SupportedMSBuildNuGetPackageVersion>
    <TargetFrameworkVersion>v4.7.1</TargetFrameworkVersion>
 </PropertyGroup>
 
 packages.config
```
```
 <packages>
  <package id="Microsoft.VisualStudio.Azure.Fabric.MSBuild" version="1.6.7" targetFramework="net471" />
</packages>
```
![pp](https://user-images.githubusercontent.com/992432/46894771-7ca5f780-ce2a-11e8-8471-e44914a8c63d.png)

**Comment by denizusky** [comment]
This turned out to be due to a strange VS2017 behavior / resetting the target architecture, and resolved by manually editing the csproj files.

**Comment by suineglive13** [comment]
Hi @nkwolf . What exactly have you edited it csproj?

**Comment by licjapodaca** [comment]
Hi @nkwolf , how did you solved this issue? can you share us the solution please...

Regards

**Comment by MattHartz** [comment]
@nkwolf What parts did you edit in your csproj files?

**Comment by senglory** [comment]
VS2019 v16.6.2, Service Fabric 7.1.417 - exactly the same issue . But when I set Service Fabric to 7.1.416 the issue's gone. How I am supposed to have it fixed for Service Fabric 7.1.417?
